main.py

```diff
@@ -145,15 +148,21 @@ def discover_delete_images(region_name, clusters):
 
         print("Number of running images found {}".format(len(running_sha)))
         ignore_tags_regex = re.compile(IGNORE_TAGS_REGEX)
+        ignore_images_regex = re.compile(IGNORE_IMAGES_REGEX)
 
         for index, image in enumerate(tagged_images):
             if image['imagePushedAt'] < time_limit or index >= IMAGES_TO_KEEP:
                 for tag in image['imageTags']:
                     if "latest" not in tag and ignore_tags_regex.search(tag) is None:
+                        # Add a condition to exclude images that contain the name regex
                         if not running_sha_set or image['imageDigest'] not in running_sha_set:
+                            image_url = repository['repositoryUri'] + ":" + tag
```

Comment:
I think we should parse only `repository['repositoryUri']` for regexp, not `repository['repositoryUri'] + ":" + tag` because the tag is already parsed by separate regexp, I don't think we should overlap them.